Core/MigrationOptions.cs

```diff
@@ -30,5 +35,40 @@ public class MigrationOptions
     public Func<Document, bool> SourceFileFilter { get; } = _ => true;
 
     public Encoding Encoding { get; } = Encoding.Default;
+
+    public IEnumerable<IDocumentConverter> Converters { get; } = new List<IDocumentConverter>();
+
+    private MigrationOptions ()
+    {
+    }
+
+    private MigrationOptions (
+        Func<Project, bool> projectFilter,
+        Func<Document, bool> documentFilter,
+        Encoding encoding,
+        IEnumerable<IDocumentConverter> converters)

```

Comment:
I think its reasonable for a library to do nothing in the case someone supplies an empty enumerable here.
It would probably be better to catch this in ConverterConsole and hide the "do nothing option" behind --dry-run or something

Reply:
`--dry-run` would make sense when specifying converters, expecting a list of changes without actually changing anything.

`MigrationOptions` without converters does not seem right to me, as a migration without any conversions enabled is not really a migration. You're right that this check should (also) be in `ConverterConsole.Program`, displaying an understandable error to the user.

Reply:
As this might be subject to change and is something that is easy to change, you could leave it as is, but add a warning or an error in `ConverterConsole.Program` for now.

Reply:
Its already a Required option for commandlineparser so it shows an error